_topic_map.yml

```diff
@@ -854,6 +854,8 @@ Topics:
     File: remove-additional-network
   - Name: Assigning a secondary network to a VRF
     File: assigning-a-secondary-network-to-a-vrf
+  - Name: Networking and GitOps
```

Comment:
But then it follows that every section of the OCP docs ought to have a _\<section> and GitOps_ section for consistency.

Maybe every section should do that; I thought GitOps might be more of a transparent thing, though.

Reply:
Let's discuss--and maybe get a little direction!  My thinking was that we'd need to either change what we had (that didn't work, it was terrible) or add new info (this PR).
Thought no. 3 is to just remove the `oc edit` step from the existing content.